src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {person.name}
+    </h2>
+
+    {person.age && (
+      <>
+        <p className="Person__age">
+          I am
+          {' '}
+          {person.age}
+        </p>
+      </>
+    )}
+
+    <p className="Person__partner">
+      {person.isMarried ? (
+        <>
+          {person.partnerName}
+          {' '}
+          is my
+          {person.sex === 'm' ? <> wife</> : <> husband</>}
+        </>
+      ) : (
+        <>I am not married</>
+      )}
```

Comment:
this looks little odd, please try to make something simpler , and you can return strings inside JSX: 
```suggestion
      {person.isMarried ? (
        <>
          {person.partnerName}
          {' '}
          is my
          {person.sex === 'm' ? 'wife' : 'husband'}
        </>
      ) : (
        'I am not married'
      )}
```